crates/cdk/src/nuts/nut04.rs

```diff
@@ -254,6 +256,7 @@ impl Default for Settings {
             unit: CurrencyUnit::Sat,
             min_amount: Some(Amount::from(1)),
             max_amount: Some(Amount::from(1000000)),
+            quote_description: true,
```

Comment:
Maybe we could return here the following depending on whether the backend supports descriptions?

``` rust
quote_description: ln.get_settings().invoice_description
```

Of course, we would need to get the value here, the code above is just a pseudo-code.

Reply:
We won't be able to do that here in `default()` so I think the default should be `false` and then [here](https://github.com/cashubtc/cdk/blob/0367e0cd4ee043a2d96f8db82de94c36737fb794/crates/cdk-mintd/src/main.rs#L153-L154) when setting `MintMethofSettings` for each backend in `mintd` we shouldn't use the default but actually set it based on what the backend supports.

Reply:
Shouldn't the default be `true` as this is the case for the vast majority of backends?

Reply:
I think false since even if a backend can support it the mint operator may not want to and it should be explicitly enabled.